jgo/jgo.py

```diff
@@ -232,17 +232,6 @@ def m2_path():
     return os.getenv("M2_REPO", (pathlib.Path.home() / ".m2").absolute())
 
 
-def expand(string, **shortcuts):
-
-    for (k, v) in shortcuts.items():
-        if string in k:
-            return "{}{}".format(
```

Comment:
ftr, fwiw, `expand` was removed because this format string expects two parameters, but only one (`v`) is given below, so this was always broken, and so very likely was not used externally.